types/types.go

```diff
@@ -103,6 +103,10 @@ const (
 	KubernetesTopologyRegionLabelKey      = "topology.kubernetes.io/region"
 	KubernetesTopologyZoneLabelKey        = "topology.kubernetes.io/zone"
 
+	KubernetesClusterAutoscalerSafeToEvictKey     = "cluster-autoscaler.kubernetes.io/safe-to-evict"
+	KubernetesClusterAutoscalerSafeToEvictEnabled = "true"
```

Comment:
`true` is very straight, so I don't think we should use another constant for that, especially the const here just implicitly means `true` actually.

Reply:
Removed.